### ISSUE TITLE
Unstoppable Error

### ISSUE THREAD

**Issue by sokos** [avocode/avocode#1992]
Screenshot http://take.ms/y8n3bT

---
* **Version**: 2.24.0
* **Sketch plugin version**: 49.0.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 50887
---
```
TypeError: Cannot read property 'get' of undefined
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:793610
    at /Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4420:31
    at List.__iterate (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:2208:13)
    at List.reduce (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/immutable/dist/immutable.js:4415:12)
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:793567)
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:793362)
    at Function.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:792232)
    at t.value (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:1751357)
    at t.value (file:///Applications/Avoc
```

**Comment by jakubzitny** [comment]
Hi @sokos, thanks for reporting this. Can you please give us some details on when/how often is this happening?

**Comment by jakubzitny** [comment]
Hi, we've just released Avocode 2.26.1, this problem should be fixed. Please update and let us know if you have any more problems 😉
